.vscode/settings.json

```diff
@@ -28,5 +28,5 @@
       */
     "editor.formatOnSaveMode": "file"
   },
-  "typescript.tsdk": "node_modules\\typescript\\lib"
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
Is this unrelated to `skipLibCheck`?

Reply:
Oops, that might be changed when I changed the typescript version in VSCode.
I always develop cactbot on a Linux machine, the backslash `\` is not compatible with POSIX paths; but vice versa, to use slash `/` on a windows machine didn't cause problems. So I think we can keep this setting as is?